dotnet/src/InternalUtilities/planning/PlannerOptions.cs

```diff
@@ -53,4 +36,10 @@ public abstract class PlannerConfigBase
     /// This configuration will be ignored if GetAvailableFunctionsAsync is set.
     /// </remarks>
     public SemanticMemoryConfig SemanticMemoryConfig { get; set; } = new();
+
+    /// <summary>
+    /// Name delimiter used to separate plugin name from function name. This should be a character or string
+    /// that is not found in any plugin or function names used by the planner.
+    /// </summary>
+    public virtual string NameDelimiter { get; set; } = "-";
```

Comment:
This needs to get propagated through to `OpenAIFunction.NameDelimiter`

Reply:
Looking more at this, it adds a bit of complexity because name delimiter would be configurable both from here and OpenAIFunction. I think more thought neesd to go into how to keep these values consistent and where the right place to configure it should be. Going to remove the option to override this setting from this PR for now.